### ISSUE TITLE
Missing test.xml file

### ISSUE THREAD

**Issue by jtorresfabra** [xarray/osgRecipes#18]
Hi @xarray ,

Just cloned the repo and noticed that  test.xml for osgeffectscompositor.cpp is not present.
https://github.com/xarray/osgRecipes/blob/master/effectcompositor/osgeffectcompositor.cpp#L140

Am I missing something?

Also CMake is not looking in the correct places for finding OSG in 64 linux distributions. I will send a pull request about that. 

Thanks!

**Comment by ghost** [comment]
> Am I missing something?

Not really, you're just supposed to pass in an effect file with --effect. Given this the default of 'test.xml' should probably be removed and replaced by an 'please provide an effect file' error or something like that. 

> Also CMake is not looking in the correct places for finding OSG in 64 linux distributions. I will send a pull request about that.

I've also had this issue as far as I recall. Did you ever send a pull request?
